lib/processing.py

```diff
@@ -0,0 +1,490 @@
+import lib.utils as utils
+import re
+import json
+import cv2
+import fitz 
+import tiktoken
+import pytesseract
+import numpy as np
+from PIL import Image
+from openai import OpenAI
+from qdrant_client import QdrantClient
+from pdf2image import convert_from_path
+from langchain_openai import ChatOpenAI
+from langchain.prompts import PromptTemplate
+from qdrant_client.http.models import Distance, VectorParams, PointStruct
+
+# def get_answer_llm(config, client_vdb, client_ia, prompt):     
+def get_answer_llm(config, data_extraction, prompt_template = 'prompt_template'):
```

Comment:
Use type hints in the function definition too, not only in the docstrings. If done correctly in combination with data classes (pydantic BaseModel or dataclass), you can avoid creating docstrings